web/src/main.js

```diff
@@ -337,6 +337,7 @@ try {
         .setData(getState().application.electricityMixMode === 'consumption'
           ? Object.values(getState().data.grid.exchanges)
           : [])
+        .setColorblindMode(getState().application.colorBlindModeEnabled)
         .render();
```

Comment:
this will trigger a `render` twice. Either remove the render from here, or remove it from the `setColorblindMode` method.